contribs/gmf/apps/mobile_alt/Controller.js

```diff
@@ -58,4 +58,21 @@ const module = angular.module('Appmobile_alt', [appBase.name, gmfControllersAbst
 
 module.controller('AlternativeMobileController', Controller);
 
+module.constant('ngeoGeolocationOptions', {
+  positionFeatureStyle: {
+    circle: {
+      radius: 6,
+      fill: {color: 'rgba(230, 100, 100, 1)'},
+      stroke: {color: 'rgba(230, 40, 40, 1)', width: 2},
+    },
+  },
+  accuracyFeatureStyle: {
+    fill: {color: 'rgba(100, 100, 230, 0.3)'},
+    stroke: {color: 'rgba(40, 40, 230, 1)', width: 2},
+  },
+  zoom: 8,
+  autorotate: true,
+  atloadingtime: true,
+});
```

Comment:
This should be defined here: https://github.com/camptocamp/demo_geomapfish/commit/8459afe6c805f4ba272ce02a12e7618a0c553a1e

Reply:
@sbrunner There is no ``vars.yaml`` file in the ngeo repo: how should I proceed then?

Reply:
No, I do it on the demo, then it's done :-)